syntax = "proto3";

import "get_user.proto";

message UpdateUserRequest {
  UserToUpdate new_user = 1;
}

message UserToUpdate {
  string id = 1;
  string username = 2;
  string email = 3;
  string password = 4;
}
